Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gableroux/unity3d:2018.3.14f1-android
```

Comment:
Maybe we can set up a separate repo for this docker image?, I guess it will not only be used in this repo going forward

Reply:
for the original one (gableroux/unity3d:2018.3.14f1-android)? or ours?

Reply:
Ours, including the android sdk , gmass and all of that

Reply:
yes sure, I've added it to the TODO, will be done before merging.